### ISSUE TITLE
Coastline is missing ice shelves

### ISSUE THREAD

**Issue by raymondben** [AustralianAntarcticDivision/SOmap#17]
Does the simplified ADD map in `quantarcticR` provide a better alternative, perhaps?

**Comment by raymondben** [comment]
WIP: `data-raw/SOmap_data.R` now has separate data objects for ice shelves/tongues/etc, and land

**Comment by Maschette** [comment]
Do I recall you finishing this? I feel like I saw it somewhere.

**Comment by raymondben** [comment]
No, not finished yet. The data objects are there, but they aren't used in the code yet. It's one of the many things I'll get back to when time allows (but anyone else can take it on in the meantime ...)

**Comment by raymondben** [comment]
Any chance we can re-define the CCAMLR planning domain data so that they do *not* include the coastline (and same for any other management layers in SOmap_data that have coastline in them)? If they are just lines over the ocean (e.g. like the IWC domains) things will be easier and faster

**Comment by mdsumner** [comment]
YESSS  - and we could add something to "bake in" a coastline at user request. sf is fast enough for that to be feasible now (though it's still not always reliable)

**Comment by Maschette** [comment]
Simple answer, Possibly? it has always been unclear about where the CCAMLR boundary starts in relation to ice shelves but in theory we could remove their coast line and use our own as long as we are quite explicit about it in the documentation. It would mean you couldn't easily use those layers for analysis like i currently do.

**Comment by mdsumner** [comment]
Well, it's the convention text that holds the boundary - but they then publish a shapefile with some unknown provenance with a baked in geography. I think it's clear where our responsibility lies, and btw of course there is this long ago thing: 

https://github.com/mdsumner/cconvention

SW even wrote a recursive descent parser to extract the coordinates from the *convention text*, but I'll leave that aside for now

I guess I'm volunteering ...

**Comment by Maschette** [comment]
My main thought would be we should have both in data so people can access if they want it, but our default would be the correct continent file.

**Comment by raymondben** [comment]
Ice shelves added as of https://github.com/AustralianAntarcticDivision/SOmap/commit/83df0f41b2bb944da60e8e255b5c36999e483e81

**Comment by raymondben** [comment]
https://github.com/AustralianAntarcticDivision/SOmap/commit/2f504df96bdd58e9eddcecbdc97158830c6e170f uses a simplified (less complexity, smaller object size) of the domains layer, but I think ideally we still want to remove the coastline from any layer except the coastline layer

**Comment by mdsumner** [comment]
I'll revisit this stuff
